src/components/Alert/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { ReactComponent as AlertIcon } from 'assets/images/icons/alert.svg';
+
+interface AlertProps {
+  errors: string[];
+  'data-test-id'?: string;
+}
+const Alert = ({ errors, ...htmlAttributes }: AlertProps): JSX.Element => (
```

Comment:
where did `htmlAttributes` come from, may I ask @manh-t 🤔 
I suggest
```suggestion
const Alert = ({ ...htmlAttributes, errors }: AlertProps): JSX.Element => (
```
to avoid unexpected override from `htmlAttributes`

Reply:
@nvminhtue `htmlAttributes` contains the attributes that cannot declare in the props, such as `data-test-id`
![image](https://github.com/manh-t/react-survey/assets/60863885/fa7cd488-9474-46ba-bb95-2c8cd5322b60)

Reply:
@nvminhtue and we cannot put the rest attributes at the beginning 😅 
![image](https://github.com/manh-t/react-survey/assets/60863885/36c51be0-d075-4c17-9cc5-39ffba1b0d87)

Reply:
ah, that's widely used as `rest` instead of `htmlAttributes` FYI @manh-t

Reply:
@nvminhtue thanks, updated it to `rest` in [d92cb3e](https://github.com/manh-t/react-survey/pull/49/commits/d92cb3e4d15bdc7b8bbf2698efaf0dc3f1a255d7)